docs/instrumenten/modelcontractbepalingen.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Modelcontractbepalingen
+toelichting: Modelbepalingen kunnen helpen om een contract op te stellen dat een organisatie in staat stelt veilige en verantwoorde algoritmen of AI-systemen in te kopen.
+levenscyclus:
+- probleemanalyse
+- ontwerp
+- dataverkenning-en-datapreparatie
+- verificatie-en-validatie
+bouwblok:
+- publieke-inkoop
+rollen:
+- aanbestedingsjurist
+- behoeftesteller
+- contractbeheerder
+- inkoopadviseur
+- projectleider
```

Comment:
Er wordt nu geen specifieke vereiste gekoppeld aan dit instrument. Wel het bouwblok Publieke Inkoop als geheel.

Ik kon namelijk zo snel niet een specifieke vereiste vinden waar dit instrument bij hoort. @ruthkoole als jij bij je review er wel eentje ziet of bedenkt, voeg gerust toe!

Reply:
hm lastig. Ik zie het ook niet zo snel. Wellicht @BartdeVisser ?

Reply:
Dit instrument kan je wel koppelen aan maatregelen. Die functionaliteit kunnen we nog een keer maken?